### DISCUSSION TITLE
Dynamic width of left hand pane in Environment screen

### DISCUSSION THREAD

**Discussion by GraemeAllanBryce** [Kong/insomnia#4227] [Ideas]
I have enclosed a screenshot of one of our environment setups. 

As you can see there are a large number and they all have relatively long names but it is not possible to see the full extent of the names in the environment edit screen. 

It would be extremely helpful if the divider in this screen was draggable or if the default width of the left hand pane was increased. 

My thanks for taking the time. 

<img width="1680" alt="2021-11-17_10-03-07" src="https://user-images.githubusercontent.com/14835538/142297554-b0f8459a-ffec-4bcc-a330-4903b22f34f2.png">

**Comment by wdawson** [comment]
Thanks for raising this @GraemeAllanBryce , I'm going to convert this to a GitHub Discussion where we have a forum for various product ideas like this.
